### DISCUSSION TITLE
Arm/Disarm Homekit voice command

### DISCUSSION THREAD

**Discussion by atluu504** [homebridge-simplisafe3/homebridge-simplisafe3#265] [General]
Hi everyone, I didn't see this in the notes and I tried different variations of trying to have homekit arm away/home and disarm the alarm via Siri. Does anyone know the exact voice commend for these actions?

TIA.

**Comment by shamoon** [comment]
Hey, I have to refer to it as “security system” or “SimpliSafe”, so like “turn off the security system”, “turn on…” sets it to away etc

**Comment by atluu504** [comment]
Hey Michael, 

I’m trying to arm home, but it seems to default to arm away no matter how I tell Siri to arm home.

**Reply by shamoon** [reply]
Hmm, I see. I have mine as part of a scene and that should do it (“hey Siri set the scene good night” etc). But sounds like a good q to ask on https://reddit.com/r/HomeKit or https://reddit.com/r/homebridge

**Comment by sterumbelow** [comment]
![image](https://user-images.githubusercontent.com/68124742/145242043-e276c796-7e01-4a69-8144-76477250b2ea.jpeg)

Simplest way is to use the default scenes suggested by the Home app for each alarm state - that'll work for Off and Away modes. Siri will accept things like: "Arm Simplisafe" or "Turn off Alarm". 

For Home mode, I just create a third scene that sets Simplisafe to Home, and then I say: "Hey, Siri. I'm at home."

![image](https://user-images.githubusercontent.com/68124742/145242454-bb0b9322-e514-4ec0-9e7e-1d86db017186.jpeg)

Further to this, you could create a shortcut (via the Shortcuts app) that activates from another command - whatever you name the shortcut is what Siri will use to run it. I.e. if it's called "Sonic the hedgehog", all you'd have to say is: "Hey, Siri. Sonic The Hedgehog." 😁

![image](https://user-images.githubusercontent.com/68124742/145243185-e9d73e35-a9a2-422d-be55-c3ffc951bd34.jpeg)

With this, you can also add a homescreen shortcut (looks like an app icon). When pressed, that shortcut will instantly run, therefore setting Simplisafe to Home. 

Hope this helps!
